src/main/java/models/Account.java

```diff
@@ -0,0 +1,45 @@
+package models;
+
+public class Account {
+    private String username;
+    private String password;
+    private String confirmPassword;
```

Comment:
this is not a property for account

Reply:
@tanmle I think it should be registerAccount